gradle/gradle-mvn-push.gradle

```diff
@@ -87,6 +87,21 @@ afterEvaluate { project ->
         }
     }
 
+    task installArchives(type: Upload) {
+        dependsOn 'build'
+        description "Installs the artifacts to the local Maven repository."
+        configuration = configurations['archives']
+        repositories {
+            mavenDeployer {
+                pom.groupId = GROUP
+                pom.artifactId = POM_ARTIFACT_ID
+                pom.version = VERSION_NAME
+
+                repository url: "file://${System.properties['user.home']}/.m2/repository"
+            }
+        }
+    }
+
```

Comment:
same nitpick here:
https://github.com/Workday/torque/pull/15#discussion_r428419579
Maybe move to the file @Pozzoooo created `upload-archives.gradle`?

Reply:
This method is actually duplicated with `uploadArchivesLocal` from `upload-archives.gradle`

Reply:
Cool I'll remove it